packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionGeneralSections/WfoSubscriptionMetadataSection.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { useTranslations } from 'next-intl';
+
+import { SubscriptionKeyValueBlock, WfoJsonCodeBlock } from '@/components';
+import { SubscriptionDetail } from '@/types';
+import { camelToHuman } from '@/utils';
+
+interface WfoSubscriptionMetadataSectionProps {
+    subscriptionDetail: SubscriptionDetail;
+}
+
+export const WfoSubscriptionMetadataSection = ({
+    subscriptionDetail,
+}: WfoSubscriptionMetadataSectionProps) => {
+    const t = useTranslations('subscriptions.detail');
+
+    const metadataBlockData = Object.entries(subscriptionDetail.metadata).map(
```

Comment:
Looks like you ony use the metadata section of the subscriptionDetail. Consider only passing that to this component

Reply:
same 

    metadata: object;